_members/jacey-curd.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Jacey Curd
+image: images/team/jacey_curd.jpeg
+description: Bio Data Scientist
+role: programmer
+aliases:
+    - Julia Curd
+    - Julia B Curd
+    - Julia Bronte Curd
+    - Jacey Bronte
+    
+links:
+  email: julia.curd@cuanschutz.edu
+  github: JaceyBronte
+---
+
+Jacey graduated with B.S. degrees in Computer Science and Biology from Duke University, where she worked in Dr. Raluca Gordan’s lab on CRISPR and MutS. In her senior thesis, she focused on the DNA binding specificity of MutS, the primary enzyme responsible for initiating mismatch repair, and possible associations with CRISPR Prime Editing technology, specifically in single base pair edits. 
```

Comment:
```suggestion
Jacey graduated with B.S. degrees in Computer Science and Biology from Duke University, where she worked in Dr. Raluca Gordan’s lab on CRISPR and MutS.
In her senior thesis, she focused on the DNA binding specificity of MutS, the primary enzyme responsible for initiating mismatch repair, and possible associations with CRISPR Prime Editing technology, specifically in single base pair edits. 
```

We write [one-sentence-per-line](https://news.ycombinator.com/item?id=31808093) in markdown format.